tools/clang/unittests/SPIRV/CodeGenSPIRVTest.cpp

```diff
@@ -372,4 +372,7 @@ TEST_F(FileTest, SpirvInterpolationError) {
   runFileTest("spirv.interpolation.error.hlsl", /*expectSuccess*/ false);
 }
 
+// Basic compute shader
+TEST_F(FileTest, BasicComputeShader) { runFileTest("empty-compute.hlsl"); }
```

Comment:
This is actually testing the `[numthreads()` attribute. So I think it's better to have it named as something like `attribute.numthreads`.

Typically it would be nice to know what the tests are from their names. `BasicComputeShader` doesn't provide enough information about its contents IMHO.

Later we will need tests for compute SV semantics and a pass through compute shader eventually as `.hlsl2spv`.